sdks/java/core/src/main/java/org/apache/beam/sdk/transforms/windowing/Window.java

```diff
@@ -486,7 +486,7 @@ public void populateDisplayData(DisplayData.Builder builder) {
           // We first apply a (trivial) transform to the input PCollection to produce a new
           // PCollection. This ensures that we don't modify the windowing strategy of the input
           // which may be used elsewhere.
-          .apply("Identity", MapElements.via(new SimpleFunction<T, T>() {
+          .apply("Identity", MapElements.<T,T>via(new SimpleFunction<T, T>() {
```

Comment:
@wtanaka, are you sure this is the actual problem you are seeing? The compiler should be able to get `T` from the argument -- explicitly specifying it shouldn't be required.

May I ask that you double check your environment? IDE version, JDK version, etc. are likely to be the cause here.